compile.sh

```diff
@@ -5,3 +5,4 @@ RUSTFLAGS="-Zlocation-detail=none" cargo +nightly build --manifest-path rust/Car
 kotlinc -include-runtime kotlin/code.kt -d kotlin/code.jar
 #kotlinc-native -include-runtime kotlin/code.kt -d kotlin/code
 dart compile exe dart/code.dart -o dart/code --target-os=macos
+dotnet publish ./csharp -c Release -o csharp/bin --self-contained
```

Comment:
`PublishSelfContained` is implied by `PublishAot`. Well, it's technically different flavour but you get the idea.

Also, since .NET 8 `publish` defaults to `Release`. We can have just a simple terse `dotnet publish ./csharp -o csharp` nowadays.